overlord/overlord.go

```diff
@@ -137,6 +138,11 @@ func New() (*Overlord, error) {
 
 	o.addManager(cmdstate.Manager(s))
 
+	o.udevMon, err = NewUDevMonitor()
```

Comment:
Please put this into a manager and create the manager here.

Reply:
Done.